frontend/src/metabase/visualizations/visualizations/SmartScalar/SmartScalar.tsx

```diff
@@ -291,7 +297,7 @@ export function SmartScalar(props) {
             </Tooltip>
 
             {canShowPreviousValue && (
-              <PreviousValue id="SmartScalar-PreviousValue" responsive>
+              <PreviousValue id="SmartScalar-PreviousValue">
```

Comment:
TypeScript couldn’t recognize this `responsive` property, nor could I.  I looked around and found nothing else using or referencing it.

Reply:
I couldn't find anything either, seems like it's safe to remove. @kamilmielnik do you remember why you added the `responsive` prop in https://github.com/metabase/metabase/pull/31722?

Reply:
It must have been a mistake: 07d086cb072aa9cb50ac7d8a4d238cadc0d2956f
Safe to remove :+1: